docs/command/mongocli-auth-login.txt

```diff
@@ -35,14 +35,6 @@ Options
      - Type
      - Required
      - Description
-   * - --cm
-     - 
-     - false
-     - Log in to Cloud Manager.
```

Comment:
remove the command or make --cm mandatory, if cm is not specified it work only for atlas

Reply:
The original command had a `isCloudManager` and `IsGov` flag and then a switch like this:
```
		switch {
		case opts.IsGov:
			opts.Service = config.CloudGovService
		case opts.isCloudManager:
			opts.Service = config.CloudManagerService
		default:
			opts.Service = config.CloudService
		}
```

But since I got rid of both `config.CloudGovService` and `config.CloudService` I changed it so that it always picks `config.CloudManagerService`. Would you rather have me get rid of the command?

Reply:
reverted